src/factory.js

```diff
@@ -118,6 +121,19 @@ export class Factory {
     return note;
   }
 
+  GraceNote(noteStruct) {
+    const note = new GraceNote(noteStruct);
+    if (this.stave) note.setStave(this.stave);
+    note.setContext(this.context);
+    return note;
+  }
+
+  GraceNoteGroup(params) {
+    const group = new GraceNoteGroup(params.notes, params.slur);
```

Comment:
should this method have a `params.options` (for consistency with the other factory methods) even though the `GraceNoteGroup` constructor's second argument is a boolean?